README.md

```diff
@@ -28,6 +29,12 @@ environment.
     make devready
     source venv/3.12/bin/activate
 
+
+Set up the `pre-commit` hook
+
+    cp ./scripts/pre-commit ./.git/hooks/
```

Comment:
Is there a reason why we're not leveraging pre-commit directly? https://pre-commit.com/

Reply:
For simple stuff like just running some ruff commands, adding another tool dependency seems like a lot. But I'd be fine with someone else adding it if they want